ansible/ansible.cfg

```diff
@@ -1,2 +1,3 @@
 [default]
 filter_plugins = ./filter_plugins
+retry_files_enabled = False
```

Comment:
This will help prevent .retry files from littering the directory. I'm not completely convinced that we get any value from the .retry values for our use case.

Reply:
:+1: